addons/account/tests/test_account_bank_statement.py

```diff
@@ -756,7 +756,7 @@ def test_statement_line_move_onchange_1(self):
         '''
 
         # Check the initial state of the statement line.
-        self.assertBankStatementLine(self.statement_line, self.expected_st_line, [self.expected_counterpart_line, self.expected_bank_line])
+        self.assertBankStatementLine(self.statement_line, self.expected_st_line, [self.expected_counterpart_line, {**self.expected_bank_line, 'amount_currency': 1250.0}])
```

Comment:
You can edit directly `expected_bank_line` inside the setupClass because this python dict is supposed to be the "default values" before any change.
The diff on `amount_currency` at https://github.com/odoo/odoo/pull/85571/files#diff-4d47f03a56193a023e415c7f4d802c8d0e264b5688f2719aea0da6da861ad4e5R1269 will no longer be necessary then